app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Just for your info: 
Some files are not good to store in GitHub repository
Let's leave it here, but some sensitive data are always added to gitignore

https://stackoverflow.com/questions/37358340/should-i-add-the-google-services-json-from-firebase-to-my-repository
(Read first and second opinion in answers)